ui/src/app/views/account-creation/component.tsx

```diff
@@ -163,69 +158,48 @@ export class AccountCreation extends
         <BoldHeader>Create your account</BoldHeader>
         <div>
           <FormSection>
-            <LongInput type='text' id='givenName' name='givenName' autoFocus
+            <TextInput id='givenName' name='givenName' autoFocus
                        placeholder='First Name'
                        value={givenName}
-                       style={(givenName.length > 80) ?
-                         inputStyles.unsuccessfulInput : inputStyles.successfulInput}
-                       onChange={e => this.updateProfile('givenName', e.target.value)}/>
-            {givenName.length > 80 &&
-            <ErrorMessage id='givenNameError'>
-              First Name must be 80 characters or less.
-            </ErrorMessage>}
+                       style={{width: '16rem'}}
+                       maxLength={80}
+                       onChange={v => this.updateProfile('givenName', v)}/>
```

Comment:
Did we get this change approved? Meaning losing the error messages

Reply:
I chatted informally with @blrubenstein about this. The thinking was that it would be just as effective to prevent the error entirely with an enforced `maxLength`.

Reply:
Ok. I just know there has been conversation around this sort of UI question before and the thinking was that to have the form "just stop working" and not indicate to the user what the problem is can be confusing. Probably more of an edge case, but just putting out what has been brought up before.

Reply:
FYI I backed out the validation change in the interest of getting this merged.